shared/app/server-config.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as File from '../util/file'
+import {serverConfigFileName} from '../constants/platform'
+import logger from '../logger'
+
+export async function updateServerConfigLastLoggedIn(username: string, serverConfig: Object) {
```

Comment:
manages writing this data from the JS side. first we read the old data and merge in the new data